### PR TITLE
fix(dispatch): clarify CDN caching

### PR DESCRIPTION
**Pull request by tripodsan**
see https://github.com/adobe/helix-pages/issues/424

### CODE REVIEW

**Comment by trieloff** [comment]
As it stands right now, this is a breaking change that disables caching on all sites, not just helix pages.

**Review by trieloff — CHANGES_REQUESTED** [review]
Don't disable caching for all sites, keep the selectivity.

**Comment by tripodsan** [comment]
> Don't disable caching for all sites, keep the selectivity.

**Comment by tripodsan** [comment]
> As it stands right now, this is a breaking change that disables caching on all sites, not just helix pages.

right now, we don't have to support any other site than helix-pages....so it doesn't really matter.....

File: src/index.js [hunk 1]
```diff
@@ -170,15 +183,13 @@ async function executeActions(params) {
         log.info('no valid response could be fetched');
       }
     }
-    // check if X-Dispatch-NoCache header is in the request,
-    // this will override the Cache-Control and Surrogate-Control
-    // response headers to ensure no caching
-    // eslint-disable-next-line no-underscore-dangle
-    if (resp && params.__ow_headers && params.__ow_headers['x-dispatch-nocache']) {
+
+    // if requested, disable caching in the CND (private) and
```
Comment (stefan-guggisberg) [inline src/index.js:187]:
```suggestion
    // if requested, disable caching in the CDN (private) and
```

**Review by trieloff — APPROVED** [review]

**Pull request merged**